src/Runner.js

```diff
@@ -48,25 +53,15 @@ export default class Runner {
     let landRequest: ?LandRequest = this.queue.dequeue();
     if (!landRequest) return;
     this.locked = true;
-    Logger.info(
-      {
-        landRequest
-      },
-      'Checking if still allowed to land...'
-    );
+    Logger.info({ landRequest }, 'Checking if still allowed to land...');
```

Comment:
Hmmm.. I need to figure out why my prettier seems to be changing these and yours isnt?